src/pages/documentation/building-mobile-extensions/index.md

```diff
@@ -0,0 +1,66 @@
+---
+description: This information can help you understand how to build your own extension.
+---
+
+# Building mobile extensions
+
+To support customer-specific features, and allow for the greatest flexibility, Adobe Experience Platform Mobile SDK provides an interface that allows customers to integrate with the Mobile Core at a much lower level. This interface allows customers to define extensions, which are similar in capabilities to the extensions that Launch has written for Adobe's internal services. The Mobile Core allows all extensions to access the events and data being sent through the Experience Platform SDK.
```

Comment:
`capabilities to the extensions that Launch has written`
I notice the usage of Launch here